src/mscorlib/corefx/System/Globalization/CultureInfo.Unix.cs

```diff
@@ -13,14 +13,20 @@ public partial class CultureInfo : IFormatProvider
         /// </remarks>
         private static CultureInfo GetUserDefaultCultureCacheOverride()
         {
-            // TODO: Implement this fully.
             return null;
         }
 
-        private static CultureInfo GetUserDefaultCulture()
+        internal static CultureInfo GetUserDefaultCulture()
         {
-            // TODO: Implement this fully.
-            return CultureInfo.InvariantCulture;
+            CultureInfo temp = null;
```

Comment:
Or does a null return from this method tell the part above the pall that there's no user default culture and it does something reasonable?